config/sync/webform.webform.contact.yml

```diff
@@ -280,21 +280,21 @@ handlers:
     settings:
       states:
         - completed
-      to_mail: _default
+      to_mail: '[webform_submission:values:category:entity:field_email:value]'
       to_options: {  }
       bcc_mail: ''
       bcc_options: {  }
       cc_mail: ''
       cc_options: {  }
-      from_mail: '[webform_submission:values:email:raw]'
+      from_mail: _default
       from_options: {  }
-      from_name: '[webform_submission:values:name:raw]'
+      from_name: ''
       reply_to: ''
       return_path: ''
       sender_mail: ''
       sender_name: ''
       subject: '[webform_submission:values:subject:raw]'
-      body: '[webform_submission:values:message:value]'
+      body: '<p>Der er kommet en ny forespørgsel med følgende info:</p><p>Navn: [webform_submission:values:name]&nbsp;</p><p>Email: [webform_submission:values:email]&nbsp;</p><p>Modtaget: [webform_submission:created]</p><p>Kategori: [webform_submission:values:category]</p><p>Emne: [webform_submission:values:subject]</p><p>Besked: [webform_submission:values:message]</p>'
```

Comment:
I think English should be the standard and Danish translations is the job of DDF to manage

Reply:
I think you are somewhat correct. I myself wasn't sure how to handle this.
The reason I ended up doing it like this is because DDF going forward are supposed to be able to create their own webforms and probably are gonna write these things in danish instead of first writing them in english and then translating.
But I'll change it to english for now.